ra-tls/ra-tls-server/lib/ra-tls-server.c

```diff
@@ -86,7 +86,7 @@ int ra_tls_server_startup(sgx_enclave_id_t id, int connd)
 	if (sgxStatus != SGX_SUCCESS || ret == -1)
 		goto err_ssl;
 
-	printf("Client: %s\n", buff);
+	printf(stdout, "Client: %s\n", buff);
```

Comment:
typo? should be fprintf.

Reply:
yes, it doesn't expose the error when compiling :(
thanks